docs/notes/tutorial_index.rst

```diff
@@ -0,0 +1,52 @@
+.. _tutorial_index:
```

Comment:
The rendered doc would look like: https://kaolin--534.org.readthedocs.build/en/534/notes/tutorial_index.html 

I'm not sure about how it's look like, also the bullet points are not appearing, could be related to: https://stackoverflow.com/questions/67542699/readthedocs-sphinx-not-rendering-bullet-list-from-rst-file .

I can fix that later (if so ping me so I make an issue)

Reply:
Fixed sphinx_rtd_theme==1.0.0